subiquity/server/controllers/filesystem.py

```diff
@@ -531,11 +531,15 @@ async def v2_edit_partition_POST(self, data: ModifyPartitionV2) \
                 and data.partition.boot != partition.boot:
             raise ValueError('edit_partition does not support changing boot')
         spec = {'mount': data.partition.mount or partition.mount}
-        if data.partition.format is not None \
-                and data.partition.format != partition.format:
+        if data.partition.format is not None:
+            if data.partition.format != partition._fs.fstype:
```

Comment:
This needs to check the original fs type not the current one I think?

Reply:
surely this time